docs/index.bs

```diff
@@ -133,6 +133,22 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
         text: obtain a storage key; url: obtain-a-storage-key
         text: storage key/equals; url: storage-key-equals
 
+spec: urlpattern; urlPrefix: https://wicg.github.io/urlpattern/
```

Comment:
After https://github.com/whatwg/urlpattern/pull/199 is merged and you make the other changes mentioned here, this section should be able to be deleted.

In general, any time something is added to this section it means you are poking into the non-exposed internals of some other spec, which is bad. It is better to export the necessary terms (like I suggest in https://github.com/whatwg/urlpattern/pull/199#pullrequestreview-1754767750), or create abstraction boundaries (like https://github.com/whatwg/urlpattern/pull/199 does with "has regexp groups" and "building a URLPattern from a Web IDL value").

Reply:
It looks like these were not exported correctly; I sent https://github.com/whatwg/urlpattern/pull/201 to fix it.

It also takes about a day for the database to update, and to get the changes locally you might need to run `bikeshed update`.